mmdet/models/dense_heads/yolact_head.py

```diff
@@ -0,0 +1,680 @@
+import mmcv
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule, xavier_init
+
+from mmdet.core import (build_sampler, fast_nms, force_fp32, images_to_levels,
+                        multi_apply)
+from ..builder import HEADS, build_loss
+from ..losses import smooth_l1_loss
+from .anchor_head import AnchorHead
+
+
+@HEADS.register_module()
+class YolactHead(AnchorHead):
+    """YOLACT box head used in https://arxiv.org/abs/1904.02689.
+
+    Note that YOLACT head is a light version of RetinaNet head.
+    Four differences are described as follows:
+    1. YOLACT box head has three-times fewer anchors.
+    2. YOLACT box head shares the convs for box and cls branches.
+    3. YOLACT box head uses OHEM instead of Focal loss.
+    4. YOLACT box head predicts a set of mask coefficients for each box.
+
+    Args:
+        num_head_convs (int): Number of the conv layers shared by
+            box and cls branches.
+        num_protos (int): Number of the mask coefficients.
+        use_OHEM (bool): If true, `loss_single_OHEM` will be used for
+            cls loss calculation. If false, `loss_single` will be used.
+    """
```

Comment:
Now we follow a strict criteria of the docstring. It is necessary to annotate every parameter and every function, but you can give a simple comment in one statement like [here](https://github.com/open-mmlab/mmdetection/blob/c342781873fd8574fe9e5be279a0bf26e0c5c5a5/mmdet/core/mask/structures.py#L186) if the function is too simple.